certified-connectors/Emigo/Readme.md

```diff
@@ -0,0 +1,48 @@
+#Title
```

Comment:
@tomaszsagra in your readme you have
```
#Title
It will enable the achievement of goals set for the sales force. It will ensure effective activities in shops, pharmacies or with medical staff. Connector Emigo will increase the effectiveness of sales teams in their daily activities.
Publisher: Sagra Technology LLC

## Prerequisites
To use the Emigo trigger integration, you must be enabled in your Emigo account. 
You will need the following to proceed:
•	An Emigo subscription.
•	An active Emigo Instance. 
•	A license Power Platform including usage of the Emigo connector
An official connector working with the Emigo Application. 

## Supported Operations
The connector supports the following operations:
•	Get all list product - this operation retrieves all list product in tenant Emigo.
•	Get all Operational Unit - this operation retrieves information about all Operation Unit.
•	Get details by Operation Unit ID – get more information about specific Operational Unit
•	Get items by table type - this operation will return the items according to the indicated value in field table type name and table name.
•	Get product by ID - this operation retrieves information about product by ID.
•	Get table name list by type - this operation will return the tables according to the indicated value in field table type name.
•	Send a message to the Operation Unit – this action that allows to send a message to specific Operational Unit
Important!  The connector Emigo supports READ ONLY. We have in planning develop some, CREATE, UPDATE and DELETE items or something data. Sagra Technology will supply new operations in year 2023. 

## Supported Triggers
Important!  The connector Emigo not supports triggers. We have in planning develop some triggers in year 2023. 

## How to get credentials
This connector uses OAuth 2.0 authentication. To sign in use your credentials of the corresponding Emigo instance. 


## Get started with your connector
This connector only supports scenarios with Emigo Sagra Technology. 

## Known issues and limitations
We do not have any known issues so far but worth noting is the connector is limited to the most important Business Unit from Emigo. 

## Common errors and remedies
We do not have any errors. 

## FAQ
Do you have a support email to contact in case of issues?
Contact this email address in case of issues with the Emigo Connector: 
Email: emigo@sagratechnology.com
Website: https://sagratechnology.com
```
Please change to 
```
#Emigo 
It will enable the achievement of goals set for the sales force. It will ensure effective activities in shops, pharmacies or with medical staff. Connector Emigo will increase the effectiveness of sales teams in their daily activities.

##Publisher: Sagra Technology LLC

## Prerequisites
To use the Emigo trigger integration, you must be enabled in your Emigo account. 
You will need the following to proceed:
•	An Emigo subscription.
•	An active Emigo Instance. 
•	A license Power Platform including usage of the Emigo connector
An official connector working with the Emigo Application. 

## Supported Operations
The connector supports the following operations:
•	Get all list product - this operation retrieves all list product in tenant Emigo.
•	Get all Operational Unit - this operation retrieves information about all Operation Unit.
•	Get details by Operation Unit ID – get more information about specific Operational Unit
•	Get items by table type - this operation will return the items according to the indicated value in field table type name and table name.
•	Get product by ID - this operation retrieves information about product by ID.
•	Get table name list by type - this operation will return the tables according to the indicated value in field table type name.
•	Send a message to the Operation Unit – this action that allows to send a message to specific Operational Unit
Important!  The connector Emigo supports READ ONLY. We have in planning develop some, CREATE, UPDATE and DELETE items or something data. Sagra Technology will supply new operations in year 2023. 

## Supported Triggers
Important!  The connector Emigo not supports triggers. We have in planning develop some triggers in year 2023. 

## How to get credentials
This connector uses OAuth 2.0 authentication. To sign in use your credentials of the corresponding Emigo instance. 


## Get started with your connector
This connector only supports scenarios with Emigo Sagra Technology. 

## Known issues and limitations
We do not have any known issues so far but worth noting is the connector is limited to the most important Business Unit from Emigo. 

## Common errors and remedies
We do not have any errors. 

## FAQ
Do you have a support email to contact in case of issues?
Contact this email address in case of issues with the Emigo Connector: 
Email: emigo@sagratechnology.com
Website: https://sagratechnology.com
```